scrapscript.py

```diff
@@ -2345,6 +2353,9 @@ def test_modulus(self) -> None:
     def test_exp_binds_tighter_than_mul(self) -> None:
         self.assertEqual(self._run("5 * 2 ^ 3"), Int(40))
 
+    def test_boolean_and_binds_tighter_than_or(self) -> None:
```

Comment:
Please add specific parser tests for precedence

Reply:
You got it